src/Settings.cpp

```diff
@@ -0,0 +1,111 @@
+#include "Settings.h"
+
+namespace weatherserver {
+
+    const utility::string_t Settings::OPC_UA_SERVER = U("opc_ua_server");
+    const utility::string_t Settings::API_OPENAQ = U("openaq_api");
+    const utility::string_t Settings::API_DARKSKY = U("darksky_api");
+    const utility::string_t Settings::PARAM_NAME_API_DARKSKY_API_KEY = U("api_key");
+    const utility::string_t Settings::PARAM_NAME_API_DARKSKY_UNITS = U("param_units");
+    const utility::string_t Settings::PARAM_NAME_API_DARKSKY_INTERVAL_DOWNLOAD_WEATHER_DATA = U("interval_download");
+
+    const std::string SETTINGS_FILE_NAME = "settings.json";
+
+    Settings::Settings(const std::string& currentDir) {
+        //default values
+        keyApiDarksky = U("");
+        units = U("si");
+        intervalWeatherDataDownload = 10;
+        port_number = 48484;
+        endpointUrl = "opc.tcp://localhost:48484";
+        hostName = "localhost";
+
+        settingsAreValid = true; //if stays true after processing - we are good, otherwise - terminate
```

Comment:
This is a bit strange: server can't work properly without API key and it is not valid with default value.